BUILD.bazel

```diff
@@ -1100,12 +1102,14 @@ cc_binary(
             "-Wl,--version-script,$(location //:src/ray/ray_version_script.lds)",
         ],
     }),
+    copts = COPTS,
     linkshared = 1,
     linkstatic = 1,
     deps = [
         "//:core_worker_lib",
         "//:src/ray/ray_exported_symbols.lds",
         "//:src/ray/ray_version_script.lds",
+        "//streaming:libstreaming_java.so",
```

Comment:
Could you explain more why add `//streaming:libstreaming_java.so` as a dependency  of  `libcore_worker_library_java.so`?

Reply:
> @chaokunyang Can you take a look at the streaming changes?
> 
> Ray core code changes look good to me. But I'm not familiar with the Windows-related build changes.

OK

Reply:
It's because I was getting a compile error regarding `#include <jni_md.h>` being missing.

Reply:
You can try same way as `libstreaming_java.so` for `jni_md `. 
I used same method as`libcore_worker_library_java.so`.  But it didn't work. So I used the method in `libstreaming_java.so` for jni. It's used in tensoflow jni too.

Reply:
I'm confused, are you saying I should add `"//streaming:jni_md.h"` to `srcs` instead, like is already the case for `libstreaming_java.so`? It doesn't work when I do that; I still get the same error.

Reply:
Do you add following?
```
    includes = [
        ".",  # needed for `include <jni.h>`
    ],
```

Reply:
In that case I get

> ERROR: `BUILD.bazel:1074:1`: in `includes` attribute of `cc_binary` rule `//:libcore_worker_library_java.so`: `'.'` resolves to the workspace root, which would allow this rule and all of its transitive dependents to include any file in your workspace. Please include only what you need. Since this rule was created by the macro `'cc_binary'`, the error might have been caused by the macro implementation

Reply:
I pushed a new commit adding a `cc_library` rule to fix this correctly.